src/snowflake/snowpark/modin/plugin/extensions/pd_overrides.py

```diff
@@ -212,33 +223,33 @@ def read_csv(
         `List of Snowflake standard compressions
         <https://docs.snowflake.com/en/sql-reference/sql/copy-into-table#format-type-options-formattypeoptions>`_ .
     thousands : str, optional
-        This parameter is not supported and will raise an error.
+       This parameter is only supported on local files.
     decimal : str, default '.'
-        This parameter is not supported and will raise an error.
+       This parameter is only supported on local files.
     lineterminator : str (length 1), optional
-        This parameter is not supported and will raise an error.
+       This parameter is only supported on local files.
     quotechar : str (length 1), optional
         The character used to denote the start and end of a quoted item. Quoted
         items can include the delimiter and it will be ignored.
     quoting : int or csv.QUOTE_* instance, default 0
-        This parameter is not supported and will raise an error.
+       This parameter is only supported on local files.
     doublequote : bool, default ``True``
-        This parameter is not supported and will raise an error.
+       This parameter is only supported on local files.
     escapechar : str (length 1), optional
-        One-character string used to escape other characters.
+       This parameter is only supported on local files.
     comment : str, optional
-        This parameter is not supported and will raise an error.
+       This parameter is only supported on local files.
     encoding : str, default 'utf-8'
         Encoding to use for UTF when reading/writing (ex. 'utf-8'). `List of Snowflake
         standard encodings <https://docs.snowflake.com/en/sql-reference/sql/copy-into-tables>`_ .
     encoding_errors : str, optional, default "strict"
-        This parameter is not supported and will raise an error.
+       This parameter is only supported on local files.
     dialect : str or csv.Dialect, optional
-        This parameter is not supported and will raise an error.
+       This parameter is only supported on local files.
     on_bad_lines : {{'error', 'warn', 'skip'}} or callable, default 'error'
-        This parameter is not supported and will raise an error.
+       This parameter is only supported on local files.
     delim_whitespace : bool, default False
-        This parameter is not supported and will raise an error.
+       This parameter is only supported on local files.
```

Comment:
Can we updated the note below to emphasize the difference between local and staged files?